plugins/dynamic-component.ts

```
@@ -0,0 +1,110 @@
+            const id = element.getAttribute("id");
+            if (!id) return;
+            const pathName = element.getAttribute("pathname");
+            const elementName = element.getAttribute("elementname");
+            const _props = element.getAttribute("props");
+            if (!pathName || !elementName)
+              throw new Error(
+                JSON.stringify({ id, pathName, elementName, props: _props })
+              );
+
+            const props = _props
+              ? (JSON.parse(decodeURI(_props)) as {})
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)